### ISSUE TITLE
offer prebuild deb files in a ppa

### ISSUE THREAD

**Issue by freddii** [o2sh/onefetch#369]
It would be useful to have prebuild deb packages in a ppa at https://launchpad.net .
So people can install it with getting updates and without having to compile it themself.

to create a deb file:
```
cargo install cargo-deb
git clone https://github.com/o2sh/onefetch
cd onefetch
cargo deb
```

**Comment by o2sh** [comment]
Actually, ppa requires source packages - that is, .dsc instead of .deb -, so binary packaging tools like  `cargo deb` don't really apply here.

I managed to publish onefetch 2.8.0 (not head) for Ubuntu focal (20.04) on amd64 ([link](https://launchpad.net/~o2sh/+archive/ubuntu/onefetch)):

```sh
sudo add-apt-repository ppa:o2sh/onefetch
sudo apt-get update
sudo apt install onefetch
```
If this doesn't match your distro version, please tell me and I'll try to push a new package. In the meantime, getting onefetch from [snap](https://snapcraft.io/onefetch) could be the right alternative.

**Comment by freddii** [comment]
yes works like a charme on 20.04 :+1: 
but maybe it can be offered for the 3 main used distributions using the snap package.

so offering it also for 18.04 and 20.10
![onefetch](https://user-images.githubusercontent.com/7213207/104088756-5077bc00-5269-11eb-8c38-4158f1305367.png)

**Comment by o2sh** [comment]
I just published two new packages: one for groovy (20.10) and one for bionic (18.04). The man page is only available in the bionic package (I forgot to do it for the other two...sorry :disappointed:).

A new entry was added in the wiki -> https://github.com/o2sh/onefetch/wiki/Installation#ubuntu-ppa

**Comment by freddii** [comment]
thanks a lot for all your work :)
